### ISSUE TITLE
[Custom Windows Event Logs integration] Error adding processors

### ISSUE THREAD

**Issue by ckauf** [elastic/integrations#7347]
Integration: [Custom Windows Event Logs](https://docs.elastic.co/integrations/winlog)
Integration version: v1.17.0
Elastic Agent version: 8.9.0
Adding a processor to the Custom Windows Event Logs integration fails with this error
```
end of the stream or a document separator is expected at line 11, column 1: - add_tags: ^
```
![image](https://github.com/elastic/integrations/assets/12035877/f11d1f42-e5a1-463f-91c2-a9995859cfe9)

**Comment by efd6** [comment]
@ckauf I don't think you are using the correct syntax. The example text in the dialogue for Custom Configuration is
```
# Winlog configuration example
#processors:
#  - drop_event.when.not.or:
#    - equals.winlog.event_id: '903'
#    - equals.winlog.event_id: '1024'
```
Note that there is a `processors:` key.

Please try
```
processors:
  - add_tags:
      tags: [web, production]
      target: "environment"
```

**Comment by ckauf** [comment]
Yes you are right. Thanks for spotting this. I think the confusion comes from other integrations that have a dedicated `processors` input box, while this has an `Advanced configuration` input.
